.github/workflows/pypi-publish.yaml

```
@@ -16,29 +16,25 @@ jobs:
-
-        python -m build
+    - name: Build the Package
+      run: |
+        cd library
+        poetry build
 
     - name: Publish to PyPI
       env:
         TWINE_USERNAME: __token__
         TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
       run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:27: Double quote to prevent globbing and word splitting

(shellcheck)